staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1175,6 +1175,51 @@ type ISCSIVolumeSource struct {
 	InitiatorName *string `json:"initiatorName,omitempty" protobuf:"bytes,12,opt,name=initiatorName"`
 }
 
+// ISCSIPersistentVolumeSource represents an ISCSI disk.
+// ISCSI volumes can only be mounted as read/write once.
+// ISCSI volumes support ownership management and SELinux relabeling.
+type ISCSIPersistentVolumeSource struct {
+	// iSCSI target portal. The portal is either an IP or ip_addr:port if the port
+	// is other than default (typically TCP ports 860 and 3260).
+	TargetPortal string `json:"targetPortal" protobuf:"bytes,1,opt,name=targetPortal"`
+	// Target iSCSI Qualified Name.
```

Comment:
Use the correct case for external variable names - `target`, `readOnly`, `portal list`.

Basically, if a human reading the API description of this field would think that you are referring to the field they are seeing in the JSON, the text should be capitalized to match JSON.

Reply:
Some of the fields are correct, some are not.

Reply:
this is a copy/paste of the ISCSIVolumeSource, follow-up and correct both